tests/Tests/500__jupyterhub/culler.robot

```diff
@@ -22,6 +22,7 @@ Verify Default Culler Timeout
     [Tags]    Sanity    Tier1
     ...       ODS-1255
     Disable Notebook Culler
+    Sleep  30s  # Give time for rollout
```

Comment:
It's not really needed in ODS-1230, since we don't care about fetching the timeout in the culler pod in that case. But I can move the sleep in the keyword to make it a bit cleaner and robust overall